tmt/steps/report/junit.py

```diff
@@ -101,7 +300,30 @@ class ReportJUnitData(tmt.steps.report.ReportStepData):
         option='--file',
         metavar='PATH',
         help='Path to the file to store JUnit to.',
-        normalize=lambda key_address, raw_value, logger: Path(raw_value) if raw_value else None)
+        normalize=tmt.utils.normalize_path)
+
+    flavor: str = field(
+        default=DEFAULT_FLAVOR_NAME,
+        option='--flavor',
+        metavar='FLAVOR',
```

Comment:
I think you can drop `metavar` - with `choices`, tmt core will turn this into `--flavor [default|custom]`, showing the actual choices.